docs/java/java-webapp.md

```diff
@@ -11,83 +11,74 @@ MetaDescription: Java web app tutorial showing how to deploy a Java web app to A
 
-## Update your project to use Azure CLI for authentication
+![Create a Web App button](images/java-webapp/create-webapp.png)
 
-Open the `pom.xml` file in your project folder (`complete`). Find and remove the snippet below in the azure-webapp-maven-plugin `configuration` section so that you will authenticate with Azure using the Azure CLI, which you did in the previous step.
+## Build and deploy to a Web App 
 
-![Remove Authentication](images/java-webapp/remove-auth.png)
+If you don't have an Azure subscription, you can sign up for a [free Azure account](https://azure.microsoft.com/pricing/free-trial/
+).
 
->**Note**: This specific configuration will now use Azure Service Principal for authentication. More details could be found at [Configure Maven to use your Azure Service Principal](https://docs.microsoft.com/azure/app-service/app-service-web-deploy-spring-boot-app-with-maven-plugin#configure-maven-to-use-your-azure-service-principal)
+<a class="tutorial-next-btn" href="https://azure.microsoft.com/pricing/free-trial/" target="_blank" style="background-color:#68217A">Create your free Azure account</a>
 
-## Build and deploy your web app to Azure
+The deploy process leverages the [Azure Account](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) extension (installed along with the Azure Functions extension as an dependency) and you need to sign in with your Azure subscription. If you do not have an Azure subscription, [sign up today](https://azure.microsoft.com//free/?b=16.48) for a free 30 day account and get $200 in Azure Credits to try out any combination of Azure services.
 
-From the command prompt or terminal window you were using earlier, rebuild the JAR file using Maven if you made any changes to the `pom.xml` file. For example:
+To log into Azure, run **Azure: Sign In** from the **Command Palette** (`kb(workbench.action.showCommands)`). You can then sign into your account using the **Device Login** flow. Click on **Copy & Open** to open your default browser.
 
-```bash
-mvn clean package
-```
+![Azure sign in code](images/java-webapp/devicelogin.png)
 
-> **Note:** You need to be in the `complete` directory when running commands in the terminal.
+Paste in the access code and continue the sign in process.
 
-Deploy your web app to Azure by using Maven:
+![Azure Device Login](images/java-webapp/devicelogin2.png)
+
+Once you have signed in, you can open the command prompt or terminal window and build the project using Maven commands. This will generate a new `name.war` or `name.jar` file in the `target` directory. 
 
 ```bash
-mvn azure-webapp:deploy
+mvn clean package
 ```
+After building the project, open the `target` directory in VS Code Explorer. Right-click on the `name.war` or `name.jar` file and choose **Deploy to Web App** button, and follow the prompts to choose the Web App for your deployment.
```

Comment:
Right-click on the `name.war` or `name.jar` file and choose **Deploy to Web App**

It is a context menu, not a button.